extensions/amp-story/0.1/amp-story.js

```diff
@@ -1004,6 +997,36 @@ export class AmpStory extends AMP.BaseElement {
   }
 
 
+  /** @private */
+  onCurrentPageIdUpdate_(pageId) {
+    this.setHistoryState_(pageId);
+  }
+
+
+  /**
+   * Save page id using history API
```

Comment:
Nit: please add a dot at the end of the comment, and the @param annotation

Reply:
done